src/python/pants/backend/docker/util_rules/docker_build_context_test.py

```diff
@@ -116,16 +124,6 @@ def test_file_dependencies(rule_runner: RuleRunner) -> None:
     )
 
     # Mixed.
```

Comment:
Stale? Not a big deal either way

Reply:
Not very informative, but it connects to the "Mixed" in the setup of `write_files()`.